src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,6 +43,13 @@ public static boolean isValidName(String test) {
     public List<String> getWordsInName() {
```

Comment:
If this function isn't used anymore perhaps it would be a better idea to remove it